texastribune/donations-app
# SMD 2016 Updates

#### What's this PR do?

Updates our top checkout on the landing page, our benefits, our videos, and our circle page.
#### Why are we doing this? How does it help us?

To get ready for our spring member drive.
The main feedback we are addressing here is:
- Making the top checkout more clearly editable and customizable to the level of giving that members would like to join at. In addition, updating the top checkout to highlight the different levels of membership we offer.
- Updating our benefits so they reflect changes to our benefits.
- Updating our videos to make them more timely.
- Updating our circle page to offer monthly/annual options on this page, to give more info about circle memberships, and to reflect the benefits changes to circle levels so that all 3 levels have the same benefits.
#### How should this be manually tested?

Visit the index page. 
- See that when the page loads, the top checkout input is autofocused with its cursor blinking. It should also now have a white bg to indicate that it's editable. 
- Adjust the donation amount to see that the JS correctly updates the current level, the amount of money to the next level, and the name of the next level. 
- Click "Explore all membership levels" and see that it takes you to the app's levels page. 
- In the right rail, see that our benefits are updated by comparing with our current benefits. Ask me to share the benefits sheet with you if you want to check the updates in more detail.
- Check out the updated checkout at different browser sizes, seeing that the changes are responsive at all sizes.
- See that the top video and image are the same. Check the new events video by clicking "Watch Now" and see that it has an updated image. Check that "Investigative projects" video is God & Governing with an updated image, and the bottom video is a look back at the lege with an updated image. Check the bottom video is the 84th Animation with an updated image. The top video's image is also an updated image of Evan now.

Visit the circle page.
- See that the circle boxes at the top of the page now have a white bg and now only have the level name, that it's a three-year commitment, and buttons for annual and monthly levels. Check these buttons to confirm that they look to the correct annual or monthly circle forms.
- See that the circle page now has a Circle member summary and then lists content, access, and recognition benefits that apply to all circle levels on the page.

Visit the membership benefits page.
- Compare to the current production page and confirm levels are updated. As above, ask me to share the benefits sheet with you if you want to check the updates in more detail.
#### Have automated tests been added?

no
#### Are there performance implications? If adding JS can it be done async? Do images/CSS/JS have cache headers set?

shouldn't be
#### What are the relevant tickets?

https://basecamp.com/1866004/projects/1175549/todos/246007467
#### How should this change be communicated to end users?

In communication with Terry, Suzanne, etc., about these updates.
#### Next steps?

Keep iterating on our donations flow.
#### Smells?

Don't think so.
#### TODOs:
- [x] Get last two videos and still images from multimedia/art and update
#### Has the relevant documentation/wiki been updated?

Didn't document, but let me know if there's anything I should that I didn't.
#### Technical debt note

More
